### ISSUE TITLE
Problems with POST request with getUsage (400 status code - bad request)

### ISSUE THREAD

**Issue by sypets** [DeepLcom/deepl-php#20]
I had some problems using this client library. Specifically, the` Translator::getUsage()` call worked on a development system, but not on other machine (with the same software package). Everything else worked very reliably.

The problem that occurs is that the `getUsage` request returns 400 HTTP status code (bad request).

I can reproduce this consistently, I even created a small test script with (basically) the same code.

After some tests I could narrow it down:

All systems use

* deeplcom/deepl v1.3.0
* PHP 7.4 or 8.1

Always worked:
* `Translator::listGlossaries()`
* `Translator::getGlossaryEntries()`
* `Translator::createGlossary()`

Problems on some machines:

* Translator::**getUsage**()

-----

The problems occured on:

* RHEL systems with PHP 8 and PHP 7 (curl 7.29.0)

Ok:

* Ubuntu with PHP 8 (curl 7.81.0)

## Suspected problem

* some differences between systems in cURL library / PHP settings (the configuration and usage of deeplcom/deepl was identical)
* `getUsage` is the only call that uses **POST** request, not GET
* with POST, the `Content-Type` header is not set and also `CURLOPT_POSTFIELDS` is not set. I assume that my different cURL libs handle that differently.

## Resproduce

It is a little difficult to reproduce because the problem does not manifest itself on all systems and I am not sure what is the relevant setting.

## Possible solution

Ideally, figure out exact reason why server sends 404. However, this also seems to fix the problem:

1. Either use **GET** with getUsage. I am not sure why POST is used, the API docs use GET: https://www.deepl.com/docs-api/general/get-usage/
2. or

in `HttpClient::sendRequest`

```
$headers['Content-Type'] = 'application/json';
$payload = json_encode( [] );
$curlOptions[\CURLOPT_POSTFIELDS] = $payload;

```

## Additional information

I activated logging and also have a test script which reproduces the problem on the affected machines and outputs some more information. I can upload the logs if requested.

The test script is here: https://github.com/sypets/sypets_test_deepl_usage_standalone

**Comment by JanEbbing** [comment]
Thanks for reporting this. `getUsage` should use a GET request, as per the API docs. I am fixing this in all client libraries.

**Comment by seekuehe** [comment]
Reopened due to no release tag yet

**Comment by JanEbbing** [comment]
Released a new version 1.4.0 that includes this fix, among some other changes, see CHANGELOG.md
